src/components/note/note.style.ts

```diff
@@ -45,11 +44,17 @@ const StyledInlineControl = styled.div`
   min-width: fit-content;
 `;
 
-const StyledTitle = styled.header`
+const StyledTitleWrapper = styled.div`
+  ${VARIANT_TYPES.map(
+    (variant) => `
+${variant}{
   font-weight: 700;
   font-size: 16px;
   line-height: 21px;
   padding-bottom: 16px;
+}
+  `
+  )}
```

Comment:
**suggestion(non-blocking)**: what do you think about setting these rules on the title wrapper itself?

This could help reduce duplication, since the rules should cascade down to any text-based HTML elements the wrapper contains.

```suggestion
  font-weight: 700;
  font-size: 16px;
  line-height: 21px;
  padding-bottom: 16px;
```

Reply:
I did try that initially, but the styling was always overridden by the underlying passed element. Targeting the actual potential elements directly allowed me to override the styling successfully 👍. I'm going to leave this is in as I think it's a cleaner approach opposed to using other importance/specificity methods.